api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/APSSResumoTrimestralCohortQueries.java

```diff
@@ -240,4 +344,191 @@ public SqlCohortDefinition getAllPatientsRegisteredInEncounterFichaAPSSANDPP() {
+    cd.addParameter(new Parameter("onOrBefore", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+
+    return cd;
+  }
+
+  private CohortDefinition getTransferredInForB10() {
+    EptsTransferredInCohortDefinition cd = new EptsTransferredInCohortDefinition();
+    cd.setName(
+        "Number of patients transferred-in from another HF during a period less than startDate B10 ");
+    cd.setProgramEnrolled(hivMetadata.getARTProgram());
+    cd.setPatientState(hivMetadata.getArtTransferredFromOtherHealthFacilityWorkflowState());
+    cd.addParameter(new Parameter("onOrAfter", "Start Date", Date.class));
+    cd.addParameter(new Parameter("onOrBefore", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+    cd.setB10Flag(true);
+
+    return cd;
+  }
+
+  private CohortDefinition getPatientsRegisteredInFichaAPSSPP() {
+    SqlCohortDefinition cd = new SqlCohortDefinition();
+    cd.setName("All Patients Registered In Encounter Ficha APSS AND PP");
+
+    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    cd.addParameter(new Parameter("location", "Location", Location.class));
+
+    Map<String, Integer> map = new HashMap<>();
+    map.put(
+        "prevencaoPositivaSeguimentoEncounterType",
+        hivMetadata.getPrevencaoPositivaSeguimentoEncounterType().getEncounterTypeId());
+    map.put("preARTCounselingConcept", hivMetadata.getPreARTCounselingConcept().getConceptId());
+    map.put("patientFoundYesConcept", hivMetadata.getPatientFoundYesConcept().getConceptId());
+    map.put("pp1Concept", hivMetadata.getPP1Concept().getConceptId());
+    map.put("pp2Concept", hivMetadata.getPP2Concept().getConceptId());
+    map.put("pp3Concept", hivMetadata.getPP3Concept().getConceptId());
+    map.put("pp4Concept", hivMetadata.getPP4Concept().getConceptId());
+    map.put("familyPlanningConcept", hivMetadata.getfamilyPlanningConcept().getConceptId());
+    map.put("pp6Concept", hivMetadata.getPP6Concept().getConceptId());
+    map.put("pp7Concept", hivMetadata.getPP7Concept().getConceptId());
+
+    String query =
+        ""
+            + "SELECT p.patient_id "
+            + "FROM patient p "
+            + "     INNER JOIN encounter e "
+            + "        ON e.patient_id = p.patient_id "
+            + "     INNER JOIN obs o "
+            + "        ON o.encounter_id=e.encounter_id "
+            + "WHERE p.voided = 0 "
+            + "    AND e.voided = 0 "
+            + "    AND o.voided = 0 "
+            + "    AND e.encounter_type = ${prevencaoPositivaSeguimentoEncounterType} "
+            + "    AND (o.concept_id = ${pp1Concept} AND o.value_coded = ${patientFoundYesConcept}) "
+            + "    AND encounter_datetime "
+            + "        BETWEEN :startDate AND :endDate "
+            + "    AND e.location_id = :location "
```

Comment:
Can't you put this query as with concept_id in (pp1,pp2....) and o.value_code=$patientfoundyes?